include/globals.h

```diff
@@ -1540,6 +1579,31 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
 #define FULL_COLOR_REMOTE_FILL 0
 #endif
 
+// Convenience shortcut to avoid cluttering all the above with #if pretzels.
+// This allows single, easy setting from either a platform.ini (or override)
+// or allowing this to be set from the command line build. e.g.:
+// $ PLATFORMIO_BUILD_FLAGS="-DLED_PIN0=14 -DUSE_ALL_NETWORKING" \
+//   pio run --target upload -e lilygo-tdisplay-s3-demo
+
+#ifdef USE_ALL_NETWORKING
```

Comment:
Please make the #ifdef an #if. Throughout the code, we interpret a flag define being "0" as meaning "no".

Reply:
We do use `#ifdef`s in quite a few places, but not to test for USE defines being "yes" or "no", which is what you're doing here. (And in case you do find a place where we do then it's legacy that should at some point be fixed.)

And I won't pre-veto a seemingly sensible use of preprocessor directives. The allergy for macros does not lie with me in the context of this particular project/GitHub org. :)